src/wolf.system/CMakeLists.txt

```diff
@@ -86,121 +102,58 @@ endif()
 
 set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 
-set (LIBRARY_TYPE "SHARED")
-
-# fetch libexpat
-message("fetching https://github.com/libexpat/libexpat.git")
-FetchContent_Declare(
-  libexpat
-  GIT_REPOSITORY https://github.com/libexpat/libexpat.git
-  GIT_TAG        master
-  SOURCE_SUBDIR  expat
-)
-
-if(WIN32 OR WIN64)
-    # fetch apr
-    message("fetching https://github.com/apache/apr.git")
-    FetchContent_Declare(
-      apr
-      GIT_REPOSITORY https://github.com/apache/apr.git
-      GIT_TAG        trunk
-    )
-endif()
-
-if(WOLF_ENABLE_CURL)
-    message("fetching https://github.com/curl/curl.git")
-    FetchContent_Declare(
-        libcurl
-        GIT_REPOSITORY https://github.com/curl/curl.git
-        GIT_TAG        master
-    )
-endif()
-
-if(WOLF_ENABLE_LUA)
-    message("fetching https://github.com/torch/luajit-rocks.git")
-    FetchContent_Declare(
-        luajit
-        GIT_REPOSITORY https://github.com/torch/luajit-rocks.git
-        GIT_TAG        master
-    )
-endif()
-
-if(WOLF_ENABLE_HTTP1_1_WS)
-    message("fetching https://github.com/madler/zlib.git")
-    FetchContent_Declare(
-        zlib
-        GIT_REPOSITORY https://github.com/madler/zlib.git
-        GIT_TAG        master
-    )
-    message("fetching https://github.com/libuv/libuv.git")
-    FetchContent_Declare(
-        libuv
-        GIT_REPOSITORY https://github.com/libuv/libuv.git
-        GIT_TAG        master
-    )
-endif()
-
-# we need wolf ssl for uSocket and apr socket
-if(WOLF_ENABLE_SSL)
-    message("fetching https://github.com/wolfSSL/wolfssl.git")
-    FetchContent_Declare(
-        wolfssl
-        GIT_REPOSITORY https://github.com/wolfSSL/wolfssl.git
-        GIT_TAG        master
-    )
-endif()
+set(LIBRARY_TYPE "SHARED")
```

Comment:
The type of library was set previously at the beginning of the file, however, it is overridden again in this line. Please remove this line.